plugins/opener/src/reveal_item_in_dir.rs

```diff
@@ -115,24 +180,36 @@ mod imp {
     target_os = "openbsd"
 ))]
 mod imp {
-
-    use std::collections::HashMap;
-
     use super::*;
+    use std::collections::HashMap;
+    use std::path::PathBuf;
 
-    pub fn reveal_item_in_dir(path: &Path) -> crate::Result<()> {
+    pub fn reveal_items_in_dir(paths: &[PathBuf]) -> crate::Result<()> {
         let connection = zbus::blocking::Connection::session()?;
 
-        reveal_with_filemanager1(path, &connection)
-            .or_else(|_| reveal_with_open_uri_portal(path, &connection))
+        reveal_with_filemanager1(paths, &connection).or_else(|e| {
+            // Fallback to opening the directory of the first item if revealing multiple items fails.
```

Comment:
I'm not sure if this fallback makes sense, is opening multiple prone to failures?

Reply:
Fallback to `reveal_with_open_uri_portal` was already used here

Reply:
Ah I see